Update exrm releases guide with latest exrm ver

sashaafm opened the pull request:
Exrm recently hit 1.0.0 (4 days ago) and got a correction yesterday (1.0.1). I noticed some time ago (have been using the exrm guides at phoenixframework.org a lot) that it still had 0.19.0 and 1.0.0-rc7. Since it hit a stable milestone, I think it's important for users to use that version.

sashaafm commented:
I've talked with @bitwalker over at `exrm` and he confirmed with me that more stuff was outdated in the guides. We agreed that I'll try to update them shortly.

ericmj commented:
Specify `~> 1.0` isntead so the guide doesn't have to update every time there's a new release.

sashaafm commented:
I think it's mostly finished. I updated deps, some explanations and all the console commands and their outputs to match the new phoenix, elixir and exrm versions.

Also added a part about the sys.config file and about using curl to check it the app is running.

sashaafm commented:
I may review my new version of the guide this week if I have time. I think I had an old version of Brunch and upon updating I believe there are more directories created when a prod release is made, but changes should be minimal.

jeregrine commented on deployment/I_exrm_releases.md:
```diff
@@ -165,88 +165,142 @@ Everything related to our releases is in the `rel/hello_phoenix` directory. Let'
--rw-r--r--   1 lance  staff  13933031 May 13 18:47 hello_phoenix-0.0.1.tar.gz
-drwxr-xr-x  26 lance  staff       884 May 13 18:47 lib
-drwxr-xr-x   5 lance  staff       170 May 13 18:47 releases
+drwxrwxr-x  6 sashaafm sashaafm 4096 Mar  1 10:29 .
```
might want to make this a generic user, unless you want everyone to find the docs when they search your name :P

jeregrine commented on deployment/I_exrm_releases.md:
```diff
@@ -297,10 +355,11 @@ Now let's try connecting a console to the running release with the `remote_conso
-Interactive Elixir (1.0.2) - press Ctrl+C to exit (type h() ENTER for help)
-iex(hello_phoenix@127.0.0.1)1>
+Interactive Elixir (1.2.0) - press Ctrl+C to exit (type h() ENTER for help)
+iex(hello_phoenix@LXLE)1> 
```
Most people I believe will have 127.0.0.1 if running this locally. May want to swap this out.

jeregrine commented:
Thanks for doing this @sashaafm couple small comments.

sashaafm commented:
Thank you for your comments @jeregrine. I'll try to update this soon with you recommendations.

sashaafm commented:
I'm sorry I took so long to update it. I just push a new commit with the recommended changes. I think it's all set. I'd like some feedback if possible :)

Thanks :+1:

jeregrine commented:
@sashaafm I am sorry I didn't merge this sooner. Could you possibly resolve the conflicts and I'll merge this ASAP.
